lib/cn/cn_tree.c

```diff
@@ -2034,18 +2191,18 @@ cn_comp_spill(struct cn_compaction_work *w)
             atomic_dec_rel(spillingp);
 
         /* Incrementing tn_ss_spilling while holding the tree lock is sufficient
-         * to keep both the route and tree node pinned across the subspill.  This
+         * to keep both the route and tree tn pinned across the subspill.  This
```

Comment:
maybe leave this reference as node? Presumably a victim of find and replace?

Reply:
Oops, yes.